src/test/java/org/opensearch/ad/e2e/RuleModelPerfIT.java

```diff
@@ -50,6 +50,21 @@ public void testRule() throws Exception {
         }
     }
 
+    public void testRuleWithDateNanos() throws Exception {
+        // TODO: this test case will run for a much longer time and timeout with security enabled
+        if (!isHttps()) {
+            disableResourceNotFoundFaultTolerence();
+            // there are 8 entities in the data set. Each one needs 1500 rows as training data.
+            Map<String, Double> minPrecision = new HashMap<>();
+            minPrecision.put("Phoenix", 0.5);
+            minPrecision.put("Scottsdale", 0.5);
+            Map<String, Double> minRecall = new HashMap<>();
+            minRecall.put("Phoenix", 0.9);
+            minRecall.put("Scottsdale", 0.9);
```

Comment:
I think it is changed to 0.6 in the main branch. Can you rebase?

Reply:
Hello, sure i have rebased and force-pushed the changes.